cmd/orchard-sidebar/main.go

```
@@ -0,0 +1,659 @@
+	fi, err := os.Stat(p)
+	return err == nil && fi.IsDir()
+}
+
+func post(query string, timeout time.Duration, out any) error {
+	body, _ := json.Marshal(map[string]string{"query": query})
+	client := &http.Client{Timeout: timeout}
+	resp, err := client.Post(graphqlURL, "application/json", bytes.NewReader(body))
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)